### PR TITLE
Prepare Storage for STG 79 beta

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

**Review by jaschrep-msft — APPROVED** [review]

**Review by amnguye — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs.ChangeFeed/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-preview.17 (Unreleased)
+## 12.0.0-preview.17 (2021-11-02)
 - Added support for service version 2020-12-06.
 - This release contains bug fixes to improve quality.
 - Fixed bug where Segment.GetCursor() would throw an ArgmentOutOfRangeException if the Segment has no Shards.
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs.ChangeFeed/CHANGELOG.md:6]: I know this isnt apart of your PR, but ArgmentOutOfRangeException get corrected to ArgumentOutOfRangeException?

Comment (seanmcc-msft) [reply sdk/storage/Azure.Storage.Blobs.ChangeFeed/CHANGELOG.md:6]: Yes I can do that

**Pull request merged**